eslint.config.js

```diff
@@ -95,24 +95,29 @@ export default tseslint.config(
         'error',
         { allow: ['arrowFunctions'] },
       ],
+      '@typescript-eslint/no-empty-object-type': [
+        'error',
+        { allowInterfaces: 'with-single-extends' }, // maybe we should turn this on in a new PR
+      ],
```

Comment:
`@typescript-eslint/ban-types` is now split into a few rules: https://typescript-eslint.io/blog/announcing-typescript-eslint-v8-beta#replacement-of-ban-types